web/styles/settings.css

```diff
@@ -1001,6 +1001,7 @@ input[type="checkbox"] {
 
     #current_bot_avatar_image {
         margin: 5px 0 8px;
+        display: none;
```

Comment:
What happens in cases of bots that actually do have an avatar? Would this not hide that image?

Reply:
Yeah this seems bugged to me. We probably instead want to just show the bot's placeholder/gravatar here, for existing bots, if one hasn't been uploaded.